pool/app/message_template/event.ml

```diff
@@ -0,0 +1,21 @@
+open Entity
+
+type update =
+  { email_subject : EmailSubject.t
+  ; email_text : EmailText.t
+  ; sms_text : SmsText.t
+  }
+[@@deriving eq, show]
+
+type event =
+  | Created of t
+  | DefaultRestored of t list
+  | Updated of t * update
+[@@deriving eq, show]
+
+let handle_event pool : event -> unit Lwt.t = function
+  | Created template -> Repo.insert pool template
```

Comment:
Add initialization of Guartian Authenticatable object (-> inserts target role to app and db)